### DISCUSSION TITLE
How to set httpOnly cookies?

### DISCUSSION THREAD

**Discussion by kamilml** [vercel/next.js#12301] [Help]
I want to make a login in my web app, using an endpoint in another server (other domain).

I can't save httpOnly cookies in the domain of my app, then I need to make something like a reverse proxy in the same server of the webapp.

Before, I did it with express, can I do it with API routes or something without express? Or maybe using serverless functions of vercel?

**Comment by rafaelalmeidatk** [comment]
Generally the reverse proxy is done at infrastructure layer (e.g. nginx) to make the requests faster (Node is not that good at this). For development you can use the experimental `rewrites` feature:

```js
module.exports = {
  experimental: {
    rewrites() {
      return [
        {
          source: '/api/:path*',
          destination: API_PATH
        }
      ];
    }
  }
};
```
